core/amber/src/main/scala/edu/uci/ics/texera/web/resource/pythonlanguageserver/PythonLanguageServerManager.scala

```diff
@@ -0,0 +1,112 @@
+package edu.uci.ics.texera.web.resource.languageserver
+
+import edu.uci.ics.amber.engine.common.AmberConfig
+
+import java.util.logging.Logger
+import scala.sys.process._
+
+object PythonLanguageServerManager {
+  private val pythonLanguageServerConfig = AmberConfig.pythonLanguageServerConfig
+  val pythonLanguageServerProvider: String = pythonLanguageServerConfig.getString("provider")
+  val pythonLanguageServerPort: Int = pythonLanguageServerConfig.getInt("port")
+  private val logger = Logger.getLogger("PythonLanguageServerManager")
+
+  // For retry
+  private val MAX_TRY_COUNT: Int = 2
```

Comment:
Can we make these constants part of the configuration so that we don't need to recompile the code when we want to change them?